cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -62,6 +63,7 @@ public class CristinMapper extends CristinMappingModule {
     public static final String UNIT_INSTITUTION_CODE = "UNIT";
     public static final ResourceOwner SIKT_OWNER = new CristinLocale("SIKT", "20754", "0", "0",
                                                                      "0").toResourceOwner();
+    private static final String NEW_LINE = "\n";
```

Comment:
There also is System.lineSeparator() or something which uses the correct separator for each system (linux, windows etc) but im not sure if its needed here.